### ISSUE TITLE
A11y: Android - Bulleted list is not read by Talk Back

### ISSUE THREAD

**Issue by stephenBarrs** [department-of-veterans-affairs/va.gov-team#10998]
Discovered by Android tester from 508 office. No one on Covid Screener team has Android device to confirm. 

Bulleted list should be announced in the following question - In the past 3 days, have you had any of these symptoms? The bulleted list was not announced.

**Comment by joshkimux** [comment]
@artsymartha68 another issue that may no longer have a team
